src/platform/forms/save-in-progress/DowntimeMessage.jsx

```diff
@@ -8,17 +7,17 @@ export default function DowntimeMessage({
   isAfterSteps,
   formConfig,
 }) {
-  const endTime = downtime.endTime;
+  const { endTime } = downtime;
   const appType = formConfig?.customText?.appType || APP_TYPE_DEFAULT;
   return (
-    <AlertBox
+    <va-alert
```

Comment:
I like that you are using the web component instead of the React binding. Thank you!